.github/CODEOWNERS

```diff
@@ -170,8 +170,9 @@
 # PRLabel: %Cognitive - Health Insights
 /sdk/healthinsights/                                                 @tomsft @koen-mertens
 
-# AzureSdkOwners: @mccoyp
 # ServiceLabel: %KeyVault
+# AzureSdkOwners: @mccoyp
+# ServiceOwners:                                                     @cheathamb36 @chen-karen @lgonsoulin
```

Comment:
Not totally sure what this linting error is, but maybe just follow the formatting on eventhubs?

https://github.com/Azure/azure-sdk-for-python/blob/07dfffc2b20773f5e9b9886a7ae8f60ee4cdea7f/.github/CODEOWNERS#L66-L71